scripts/meta

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python
+# vim: noexpandtab:tabstop=4:shiftwidth=4:fileencoding=UTF-8
+#
+# @author Sebastian Bachmann <sebastian.bachmann@tuwien.ac.at>
+# @desc This script is used to organize scripts
+# @date 10.05.2016
+# @tags
+# @system all
+#
+# Comments in the Beginning of a script define some things and are parsed by this meta script
+# The aim of this script is to find the right script in no time!
+#
+# The Metatag System of Epydoc is used (http://epydoc.sourceforge.net/manual-fields.html#module-metadata-variables)
+# and extendet with the following special tags:
+# 
+# MetaTag					Meaning
+# ----------------------------------------------------------
+# @date						The Date when the script was created, use DD.MM.YYYY as format
+# @desc						A Short Description (One Line!)
+# @author					Name of the Author and email in < >
+# @tags						Comma Seperated List of Tags, there is a list of common tags below
+# @system					Name of the System that the script is intended for (comma seperated)
+# @depends					Other script names that depend on (the output) of this script
+
+metatags = ["date", "desc", "author", "tags", "system", "depends"]
+
+# 
+# Because there are not only python scripts, we use the @ declaration, inside the comment.
+# Beware, that these metatags should be applied at the beginning of the file (in the first 10 rows)
+#
+# If you create own Tags, please document them here! 
+tags = {'configure': "This script is used for the build configuration",
```

Comment:
again tab usage does not align (but should be used here), can you switch to our defaults as given in .clang-format